scripts/build/termux_step_install_pacman_hooks.sh

```diff
@@ -0,0 +1,30 @@
+termux_step_install_pacman_hooks() {
+	[ "$TERMUX_PACKAGE_FORMAT" != "pacman" ] && return
+
+	# Installing hooks
+	local hooks=$(find $TERMUX_PKG_BUILDER_DIR -mindepth 1 -maxdepth 1 -name "*.alpm.hook")
```

Comment:
You should move assigning of variable to separate statement, see [SC2155](https://www.shellcheck.net/wiki/SC2155).
Also you should explicitly quote this.

```suggestion
	local hooks; hooks="$(find $TERMUX_PKG_BUILDER_DIR -mindepth 1 -maxdepth 1 -name "*.alpm.hook")"
```

Reply:
Probably it will be better to do 
```
	readarray -t hooks < <(find $TERMUX_PKG_BUILDER_DIR -mindepth 1 -maxdepth 1 -name "*.alpm.hook")
```
And later use it like 
```
"${hooks[@]}"
```